bot/cogs/groups.py

```diff
@@ -0,0 +1,68 @@
+from discord import Embed, NotFound
+from discord.utils import find
+from discord.ext import commands
+
+from common.bot_logger import get_logger
+from common.helper_functions import reply_and_delete
+
+logger = get_logger(__name__)
+
+
+class GroupCog(commands.Cog, name="Group management"):
+    def __init__(self, bot):
+        self.bot = bot
+        logger.info("Group cog initialised")
+
+    @commands.group(
+        name="group",
+        invoke_without_command=True,
+        help="Get information about groups",
+    )
+    async def group(self, ctx):
+        await ctx.send("No subcommand was found!")
+        await ctx.send_help(self.member)
+
+    @group.command(help="Outputs all users with the role given in the argument(s)")
+    async def getgroup(self, ctx, *args):
+        try:
+            if ctx.guild:
+                roles = ctx.guild.roles
+                for arg in args:
+                    if arg.lower() in map(lambda r: r.name.lower(), roles):
+                        # get members and pass info
+                        # pylint: disable=cell-var-from-loop
+                        role = find(lambda r: r.name.lower() == arg.lower(), roles)
+                        # pylint: enable=cell-var-from-loop
+
+                        msg = Embed(title=role.name, colour=0xE62272)
```

Comment:
Maybe we should have a python file in common for constants such as the colours found in Thalia's style guide

Reply:
Yes